src/write_gcm.c

```diff
@@ -1093,9 +1089,10 @@ static int wg_oauth2_get_auth_header_nolock(oauth2_ctx_t *ctx,
 static int wg_oauth2_talk_to_server_and_store_result(oauth2_ctx_t *ctx,
     const char *url, const char *body, const char **headers, int num_headers,
     cdtime_t now) {
-  char response[2048];
-  if (wg_curl_get_or_post(response, sizeof(response), url, body,
-      headers, num_headers, 0) != 0) {
+  char* response = NULL;
+  if (wg_curl_get_or_post(&response, url, body, headers, num_headers, 0)
+      != 0) {
```

Comment:
Should fit on the previous line.

Reply:
I tried, it sits at 81 characters :(

Reply:
Sigh. You're right, I didn't notice the `{` that snuck onto the next line.